crypto/md5/asm/md5-loongarch64.pl

```diff
@@ -0,0 +1,287 @@
+#! /usr/bin/env perl
+# Author: Min Zhou <zhoumin@loongson.cn>
+# Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+# Reference to crypto/md5/asm/md5-x86_64.pl
+# MD5 optimized for LoongArch.
+
+use strict;
+
+my $code;
+
+my ($zero,$ra,$tp,$sp,$fp)=map("\$r$_",(0..3,22));
+my ($a0,$a1,$a2,$a3,$a4,$a5,$a6,$a7)=map("\$r$_",(4..11));
```

Comment:
one of the longarch documents references a0-a7 as parameter passing registers.. Is r4-r11 actually defined as param passing somewhere in the spec?

Reply:
https://github.com/loongson/la-abi-specs/blob/release/lapcs.adoc#general-purpose-registers

a0-a7 are aliases of r4-r11.

Reply:
Yes, thanks!